src/panels/lovelace/editor/config-elements/hui-card-features-editor.ts

```diff
@@ -78,6 +79,7 @@ const UI_FEATURE_TYPES = [
   "cover-position",
   "cover-tilt-position",
   "cover-tilt",
+  "date",
```

Comment:
This feels a bit too generic. I think we should rename it to "set-date". It's also what the button says

Reply:
Or "date-input"

Reply:
Maybe "date-set" so it matches the <domain>-X style?